canonicalwebteam/templatefinder/templatefinder.py

```diff
@@ -25,6 +26,22 @@ class WebteamBlockLexer(BlockLexer):
     )
 
 
+class IDRenderer(Renderer):
+    def header(self, text, level, raw=None):
+        header_id = (text.replace(" ", "-")).lower()
+        header_id = re.sub("[^\w-]", "", header_id)
+        header_id = re.sub('<[^<]+?>', '', header_id)
```

Comment:
Does it work here? If we want to remove `<...>` html tags shouldn't it be done before we remove special chars line before it?

Reply:
lol. it does indeed not work. and that is clearly why =)